lntest/harness.go

```diff
@@ -573,6 +576,42 @@ func (n *NetworkHarness) StopNode(node *HarnessNode) error {
 	return node.stop()
 }
 
+//SaveProfilesPages hits profiles pages of all active nodes and writes it to disk
+//using a similar naming scheme as to the regular set of logs.
+func (n *NetworkHarness) SaveProfilesPages() {
+	for _, node := range n.activeNodes {
+		url := "http://localhost:%d/debug/pprof/goroutine?debug=1"
+
+		port := 9000 + node.NodeID
```

Comment:
IMHO, this port should be kept as part of the node structure and not recalculate here

Reply:
Done.